proto/metadata.proto

```diff
@@ -14,3 +14,8 @@ option (gogoproto.unmarshaler_all) = true;
 message InUnionMetadata {
     required bool in_union = 1 [(gogoproto.nullable) = false];
 }
+
+message CompareInMetadata {
+    required bool has_null = 1 [(gogoproto.nullable) = false];
+    required bytes datums = 2;
```

Comment:
how about call it `not_null_datums`

Reply:
looks like this should be optional? (or you need to add `(gogoproto.nullable) = false`)